src/amo/reducers/home.js

```diff
@@ -226,19 +230,12 @@ type LoadHomeDataAction = {|
 |};
 
 export const loadHomeData = ({
-  collections,
-  heroShelves,
-  shelves,
+  homeShelves,
 }: LoadHomeDataParams): LoadHomeDataAction => {
-  invariant(collections, 'collections is required');
-  invariant(shelves, 'shelves is required');
```

Comment:
I think we should keep an `invariant` for `homeShelves`. Is there ever a time where we'd be calling `loadHomeData` and `homeShelves` is falsey?